apps/darts-modernisation/darts-gateway/demo.yaml

```diff
@@ -10,6 +10,7 @@ spec:
       ingressHost: darts-gateway.demo.platform.hmcts.net
       environment:
         DARTS_LOG_LEVEL: TRACE
```

Comment:
Should we set this to INFO level as well, so that it matches prod? We only wanted to enable trace for soap requests which is now covered in your other variable.

Reply:
I agree it would make sense to align the two I have updated the PR :)